python/tests/transformer.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+import unittest
+
+import numpy as np
+from equistore import TensorBlock, TensorMap
+from skcosmo.feature_selection import FPS as FPS_f
```

Comment:
why do you rename FPS to FPS_f?

Reply:
I forgot to rename it from the time I thought it would be necessary to make a sample selection too. I'll rename it